.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,10 +8,12 @@ to the issue. -->
 
 - [ ] Commit sequence broadly makes sense and commits have useful messages
 - [ ] New tests are added if needed and existing tests are updated
-- [ ] When applicable, versions are updated in `.cabal` and `CHANGELOG.md` files according to the
+- [ ] All visible changes are prepended to the latest section of a `CHANGELOG.md` for the affected packages.
+      **_New section is never added with the code changes._** (See [RELEASING.md](https://github.com/intersectmbo/cardano-ledger/blob/- [ ] When applicable, versions are updated in `.cabal` and `CHANGELOG.md` files according to the
       [versioning process](https://github.com/intersectmbo/cardano-ledger/blob/master/RELEASING.md#versioning-process).
-- [ ] The version bounds in `.cabal` files for all affected packages are updated. **If you change the bounds in a cabal file, that package itself must have a version increase.** (See [RELEASING.md](https://github.com/intersectmbo/cardano-ledger/blob/master/RELEASING.md#versioning-process))
-- [ ] All visible changes are prepended to the latest section of a `CHANGELOG.md` for the affected packages. **New section is never added with the code changes.** (See [RELEASING.md](https://github.com/intersectmbo/cardano-ledger/blob/master/RELEASING.md#changelogmd))
+- [ ] The version bounds in `.cabal` files for all affected packages are updated.
+      **_If you change the bounds in a cabal file, that package itself must have a version increase._** (See [RELEASING.md](https://github.com/intersectmbo/cardano-ledger/blob/master/RELEASING.md#versioning-process))
+master/RELEASING.md#changelogmd))
```

Comment:
I think this is the orphaned text

Reply:
Not quite. I just moved the **bold** section to a new line, so it is more visible

Reply:
```suggestion
```

Reply:
You are absolutely right. I have no idea why it was that hard to move this thing around :facepalm: